### PR TITLE
geoJSON BoundingBox - Completely refactored

### PR DESCRIPTION
**Pull request by rsupnekar**
Bounding box is not used in EMP feature.

This fixes the timestamp and timespan properties

### CODE REVIEW

File: sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonParser.java [hunk 1]
```diff
@@ -66,9 +76,8 @@
                     for (JsonValue geometry : geometryArray) {
                         parseGeometry(featureList, geometry.asObject(), null);
                     }
-                    for (GeoJsonFeature feature : featureList) {
-                        feature.setProperties(new GeoJsonProperties());
-                    }
+                } else if (objectType.equals("bbox")) {
+
```
Comment (ee11cbb19052e40b07aac0ca060c23ee) [inline sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonParser.java]: maybe log a warning message here

File: sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonParser.java [hunk 2]
```diff
@@ -79,97 +88,124 @@
...
+                        geoTimeSpan.setEnd(zonedDate.parse(timeSpanEnd));
+                        feature.getTimeSpans().add(geoTimeSpan);
+                    }
+                    String timeStamp = timePrimitive.get("timeStamp").asString();
+                    if (timeStamp != null && !timeStamp.isEmpty()) {
+                        Date ts = zonedDate.parse(timeStamp);
+                        feature.setTimeStamp(ts);
+                    }
+                } catch (java.text.ParseException e) {
+                    e.printStackTrace();
```
Comment (ee11cbb19052e40b07aac0ca060c23ee) [inline sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonParser.java]: This methods throws a ParseException, why catch it here?

**Review by ee11cbb19052e40b07aac0ca060c23ee — CHANGES_REQUESTED** [review]
see comments

File: sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonParser.java (hunk 2 above)
Comment (Ish-rivera) [reply sdk/sdk-api/src/main/java/mil/emp3/json/geoJson/GeoJsonParser.java]: Same.

**Review by ee11cbb19052e40b07aac0ca060c23ee — APPROVED** [review]

**Comment by mubinakhan** [comment]
https://github.com/orgs/missioncommand/projects/3
________________________________

**Comment by mubinakhan** [comment]
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tests:test-basic:compileDebugJavaWithJavac'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

**Comment by mubinakhan** [comment]
screenshot_20170207-143315 1

**Comment by mubinakhan** [comment]
Reintegrate

**Comment by mubinakhan** [comment]
screenshot_20170208-151341 1
screenshot_20170208-151420 1

**Pull request merged**